lib/interfaces/interface.ts

```
@@ -0,0 +1,33 @@
+import React from "react";
+
+// FCWC: FunctionComponentWithChildren
+export type FCWC<T> = React.FC<React.PropsWithChildren<T>>;
+
+// Propless: Component without props
+export type Propless = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)